src/modules/billing/jobs/create-charge.js

```diff
@@ -63,35 +63,37 @@ const getTransactionStatus = batch => get(batch, 'invoices[0].invoiceLicences[0]
 const handler = async job => {
   batchJob.logHandling(job);
   const transactionId = get(job, 'data.billingBatchTransactionId');
+  const batchId = get(job, 'data.batchId');
 
   // Create batch model from loaded data
   const batch = await transactionsService.getById(transactionId);
```

Comment:
add comment in here to explain that the batch is batch and all lower level related objects for pre-sroc but for sroc we don't map all the related objects that is not needed and will slow down performance